packages/react-app/src/components/ControlTools.jsx

```diff
@@ -0,0 +1,286 @@
+import React, { useState, useEffect } from "react";
```

Comment:
Is this basically the same as the old DrawTools file? Looks similar, but it's hard for me to see the changes in the UI.

Reply:
Yes, I think in scaffolding I merged two files into this one (including DrawTools) within.